packages/terra-framework-docs/src/terra-dev-site/doc/compact-interactive-list/Examples.2/NumberOfColumns.5.doc.mdx

```diff
@@ -0,0 +1,23 @@
+import { Notice } from "@cerner/terra-docs";
+import WidthBreakpoints from './WidthBreakpoints?dev-site-examples';
+
+# Number of Columns
+
+### Description
+
+* The [_numberOfColumns_](/components/cerner-terra-framework-docs/compact-interactive-list/about#compact-interactive-list-props-table) prop is used to arrange the list data in visual columns. 
+* If not explicitly set by user, the [_numberOfColumns_](/components/cerner-terra-framework-docs/compact-interactive-list/about#compact-interactive-list-props-table) defaults to 1.
+
+### Change the Number of Columns dynamically
+
+This example demonstrates a [Compact Interactive List](/components/cerner-terra-framework-docs/compact-interactive-list/about) being placed in the parent container, which uses the width breakpoints to change the [_numberOfColumns_](/components/cerner-terra-framework-docs/compact-interactive-list/about#compact-interactive-list-props-table) prop depending on the screen width. 
+Change the browser window width to see the number of columns change.
+
+<WidthBreakpoints />
+
+<Notice variant="caution" ariaLevel="2">
+
+With vertical flow the list items are distributed evenly between all the columns but the last one. The last column accommodates the reminder after the number of items is divided by number of columns.
```

Comment:
```suggestion
With vertical flow,  the list items are distributed evenly between all the columns but the last one. The last column accommodates the remainder after the number of items is divided by number of columns.
```

Reply:
Great suggestion, and much appreciated, but it looks like all wording and styling comment were outdated per this commit by @cldeitrich: https://github.com/cerner/terra-framework/pull/1950/commits/62c72c0098e6edb05ec923b22cf997650ac247bf